platform/mac/Credits.rtfd/TXT.rtf

```diff
@@ -4,5 +4,5 @@
 \margl1440\margr1440\vieww12600\viewh10200\viewkind0
 \pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\qc\partightenfactor0
 
-\f0\b\fs24 \cf0 Copyright \'a9 2020- Solar2D\
+\f0\b\fs24 \cf0 Copyright \'a9 2020- Solar2D.\
```

Comment:
Hi :) It seems to me that the dash after "2020" should be removed.

Reply:
Fixed